### PR TITLE
This pull request is to update the main repo with 3 updates

### PR DESCRIPTION
**Pull request by mxaddict**
Added @var documentation for document variables
Fixed type guessing for booleans
Fixed the type regex to be case insensitive (Cause PHP allows both UPPER and lower case version)
Added support for Class/Interface multiline syntax
Added support for multiple inheritance (PHP Allows an interface to extend multiple interfaces/classes)

### CODE REVIEW

File: autoload/pdv.vim [hunk 1]
```diff
@@ -59,13 +59,16 @@ let s:regex["attribute"] = '^\(\s*\)\(\(private\s*\|public\s*\|protected\s*\|sta
 " [:spacce:]*(abstract|final|)[:space:]*(class|interface)+[:space:]+\(extends ([:identifier:])\)?[:space:]*\(implements ([:identifier:][, ]*)+\)?
 let s:regex["class"] = '^\(\s*\)\(\S*\)\s*\(interface\|class\)\s*\(\S\+\)\s*\([^{]*\){\?$'
 
+let s:regex["variable"] = '^\(\s*\)\(\$[^ =]\+\)\s*=\s*\([^;]\+\);$'
+let s:regex["newobject"] = '^\s*new\s*\([^(;]\+\).*$'
+
 let s:regex["types"] = {}
 
-let s:regex["types"]["array"]  = "^array *(.*"
+let s:regex["types"]["array"]  = '^array *(.*\c'
```
Comment (tobyS) [inline autoload/pdv.vim]: Please use \s* for whitespaces. Adding the \c option is a good idea.

File: autoload/pdv.vim [hunk 2]
```diff
@@ -146,7 +152,20 @@ func! s:ApplyIndent(text, indent)
 endfunc
 
 func! pdv#ParseClassData(line)
+	let l:i = 1
+	let l:curlys = []
 	let l:text = getline(a:line)
+	let l:curlys = matchlist(l:text, '^.*\({\).*$')
+	while l:i <= 10
```
Comment (tobyS) [inline autoload/pdv.vim]: I don't get what this loop actually does. Could you add a comment or even better move it to a function?

File: templates_snip/attribute.tpl [hunk 3]
```diff
@@ -1,5 +1,3 @@
 /**
- * ${1:{{name}}}
- *
- * @var ${2:{{type}}{{^type}}mixed{{/type}}}
+ * @var ${2:{{type}}{{^type}}mixed{{/type}}} ${1:{{name}}}
```
Comment (tobyS) [inline templates_snip/attribute.tpl]: Please leave the default templates as they are. You can overwrite the defaults in your installation through a dedicated template directory.

**Comment by tobyS** [comment]
Thanks for the additions. Looks pretty good. I left some small remarks.

**Comment by mxaddict** [comment]
Hi @tobyS, I'm making changes to the commits, and will create a new merge/pull request after I'm done,

BUT, I was planning on making some new TEST for the added functionality that I did for class/method/variable declaration parsing, how Do I run the current test suite? Cause when I run the runTests.sh file I get an error about a missing file.

**Comment by mxaddict** [comment]
I don't think this is needed anymore.

**Pull request closed without merging**